google_compute_engine/logger.py

```diff
@@ -32,6 +32,7 @@ def Logger(name, debug=False, facility=None):
   """
   logger = logging.getLogger(name)
   logger.handlers = []
+  logger.addHandler(logging.NullHandler())
```

Comment:
Thanks for catching that! The library does need to support 2.6 (I expect the 2.6 test to fail).